### PR TITLE
Add endpoint for swagger API

### PR DESCRIPTION
**Pull request by nuzil**
## This PR is a:

- [ ] New topic
- [x] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

I added small section to Magento 2.3 docs related to Swagger URL that is available for Async API.

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by osrecio** [comment]
Hi @nuzil thanks for your contribution. 

Only a question: Bulk API is available for Magento 2.1 and Magento 2.2 ?

Thanks!

**Comment by nuzil** [comment]
@osrecio Only for 2.2 Commerce, but swagger was not ported there, so this changes are only for 2.3

**Comment by osrecio** [comment]
Thanks for the clarification @nuzil. In that case. You should copy the content from:
https://github.com/magento/devdocs/blob/master/guides/v2.1/rest/generate-local.md
to 
https://github.com/magento/devdocs/blob/master/guides/v2.3/rest/generate-local.md
and add your changes. 

If you do not do that, the changes will be reflected in 2.1, 2.2 and 2.3 when should only be for 2.3

**Comment by keharper** [comment]
@nuzil, @osrecio raises a good point. Which versions of Magento do these Swagger customizations apply to? For sure, you'll need to break one of these symbolic links:

* 2.1 > 2.2, if the Swagger enhancements have been backported to 2.2
* 2.2 > 2.3, if they apply only to 2.3

Also, if this new feature applies to 2.3.1 (and its 2.2.x equivalent), please use the newly-created `2.3.1-integration` branch instead of master.

File: guides/v2.1/rest/generate-local.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ The generated Swagger documentation provides the capability to test REST request
 {:.bs-callout .bs-callout-info}
 The **Try it out!** feature is not available on Swagger documentation published on the Magento devdocs website.
 
+### REST reference for Asynchronous API
+
+In addition to standard Swagger you can also generate live REST Asynchronous API documentation. To create this documentation, add `?type=async` parameter in standard Swagger URL:
```
Comment (keharper) [inline guides/v2.1/rest/generate-local.md]:
```suggestion
You can also use Swagger to generate live asynchronous API REST documentation. To create this documentation, add the `?type=async` parameter to the standard Swagger URL: 
```

File: guides/v2.1/rest/generate-local.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ The generated Swagger documentation provides the capability to test REST request
 {:.bs-callout .bs-callout-info}
 The **Try it out!** feature is not available on Swagger documentation published on the Magento devdocs website.
 
+### REST reference for Asynchronous API
+
+In addition to standard Swagger you can also generate live REST Asynchronous API documentation. To create this documentation, add `?type=async` parameter in standard Swagger URL: 
+
+`http://<magento_host>/swagger?type=async`
+
+This will return you information about all resources available for Asynchronous REST API.
```
Comment (keharper) [inline guides/v2.1/rest/generate-local.md]:
```suggestion
Swagger returns information about all resources available for asynchronous REST APIs.
```

**Comment by keharper** [comment]
Oh awesome. I'll get our Twitter master to send out a tweet about it.

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks so much, @nuzil

**Pull request merged**